code/scripting/ade.cpp

```diff
@@ -626,10 +634,21 @@ void ade_table_entry::OutputMeta(FILE *fp)
 					fprintf(fp, "<dd>%s</dd>\n", Description);
 				}
 
-				//***Result: ReturnDescription
-				if(ReturnDescription != NULL) {
-					fprintf(fp, "<dd><b>Returns:</b> %s<br>&nbsp;</dd>\n", ReturnDescription);
-				} else {
+			    if (DeprecationVersion.isValid()) {
+				    if (DeprecationMessage != nullptr) {
+					    fprintf(fp, "<dd><b>Deprecated starting with version %d.%d.%d:</b> %s</dd>\n",
+					            DeprecationVersion.major, DeprecationVersion.minor, DeprecationVersion.build,
+					            DeprecationMessage);
+				    } else {
+					    fprintf(fp, "<dd><b>Deprecated starting with version %d.%d.%d.</b></dd>\n",
+					            DeprecationVersion.major, DeprecationVersion.minor, DeprecationVersion.build);
+				    }
+			    }
+
+			    //***Result: ReturnDescription
+			    if (ReturnDescription != NULL) {
```

Comment:
I'm not sure you meant to change these lines (they now have mixed tab/space indentation), but if you did, this `NULL` should become a `nullptr` to avoid clang-tidy getting mad.